app/component/map/popups/DynamicParkingLotsPopup.js

```diff
@@ -57,7 +57,7 @@ class DynamicParkingLotsPopup extends React.Component {
       );
     }
 
-    return <div className="padding-vertical-small">{text}</div>;
```

Comment:
The CSS class is necessary for the styling.

![Screenshot from 2020-07-27 10-43-55](https://user-images.githubusercontent.com/151346/88522058-218cc700-cff6-11ea-9be3-803de418a48b.png)
![Screenshot from 2020-07-27 10-43-48](https://user-images.githubusercontent.com/151346/88522065-22255d80-cff6-11ea-98bc-42bcffb1272b.png)

Can you find a solution for this?

Reply:
I do not see any difference now, so please check the latest updates, maybe it fixed this (however it wasn't the main purpose..) If you still have this issue, I'll look for another solution.

Reply:
It's the padding of "3 von 24 Stellplätzen geöffnet".

Reply:
![Screenshot from 2020-07-29 09-49-10](https://user-images.githubusercontent.com/151346/88772289-d7841c80-d180-11ea-98bd-f54f4d352819.png)

Just checked it and the padding is not restored.

Reply:
Ok, I see, check now pls!